index.html

```diff
@@ -229,5 +244,60 @@ <h2 class="spaced">About me</h2>
                 </div>
             </div>
         </section>
+
+        <footer>
+        <div class="contact-form">
+            <p class="p-contact">I'm always interested in hearing about 
+                new projects, so if you'd like to chat 
+                please get in.</p>
+            
+        <form id="contact-me" action="https://formspree.io/f/contact-me" method="post">
```

Comment:
- [ ] Good job implementing formspree 👏  I think there is something wrong with your form endpoint. In order for you to receive the data in the forms, you should have your own form-id provided by formspree. You should consider double-checking it. This is what it's supposed to look like:

![image](https://user-images.githubusercontent.com/66967274/145031481-44d63d9b-2cac-4ec9-a502-1efc83822652.png)